tg/MainScreenListOfGames/TableListOfGameCell.swift

```diff
@@ -46,6 +55,8 @@ class TableListOfGameCell: UITableViewCell {
         guard let released = game.released, let rate = game.rating else {return}
         rating.text = "\(rate)" //String(rate)
         year.text = dateFormatter(released)
+        
+        cellDesign()
```

Comment:
It's incorrect place for changing design of cell. This method is called each time when cell appears on the screen (scrolling, reloading data)